src/__tests__/utils.tsx

```diff
@@ -62,16 +62,21 @@ function wrap({ default: Component }: { default: React.ComponentType }, Wrapper:
 
 export function requireComponent(componentName: string): any {
   if (componentName === 'split-panel') {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
```

Comment:
Had to add eslint disable comments for the `require` statements as I touched this file. I have no quick-and-easy alternatives for this, as sync `import` does not support dynamic (i.e, non hard-coded) paths. Async imports could be a solution, but this requires changes in all tests that consume these functions.